deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -341,9 +396,31 @@ def areAgent_processes_active(agent_params):
         Returns:
             str: Os name.
         """
-        return bool([int(numero) for numero in Executor.execute_command(agent_params, 'pgrep wazuh').splitlines()])
+        os_type = HostInformation.get_os_type(agent_params)
+
+        if os_type == 'linux':
+            result = ConnectionManager.execute_commands(agent_params, 'pgrep wazuh')
+            if result.get('success'):
+                return bool([int(number) for number in result.get('output').splitlines()])
+            else:
+                return False
 
-    def isAgent_port_open(inventory_path):
+        if os_type == 'macos':
+            result = ConnectionManager.execute_commands(agent_params, 'pgrep wazuh')
+            if result.get('success'):
+                return bool([int(number) for number in result.get('output').splitlines()])
+            else:
+                return False
+
+        elif os_type == 'windows':
+            result = ConnectionManager.execute_commands(agent_params, 'Get-Process -Name "wazuh-agent" | Format-Table -HideTableHeaders  ProcessName')
+            if result.get('success'):
+                return 'wazuh-agent' in result.get('output')
+            else:
+                return False
+
+
+    def is_agent_port_open(inventory_path):
```

Comment:
Decorate this method as `@staticmethod`

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/5310/commits/b1712ec86ca12fdc5c8469f100bdd7b487838276